packages/spatial/src/camera/components/CameraComponent.ts

```diff
@@ -40,13 +41,23 @@ export const CameraComponent = defineComponent({
     camera.cameras = [new PerspectiveCamera().copy(camera, false)]
     return camera
   },
-  onSet: (entity, component, json: undefined) => {
+  onSet: (entity, component, json) => {
     addObjectToGroup(entity, component.value as ArrayCamera)
```

Comment:
please add object in a useEffect not in onSet

Reply:
implemented